public/locales/en/common.json

```diff
@@ -1,6 +1,57 @@
 {
-  "home": "Home",
   "contact": "Contact",
   "copyright": "Copyright",
-  "support": "Support"
+  "support": "Support",
+  "es": "Spanish",
+  "en": "English",
+  "routes": {
+    "/": "Home",
+    "voters": "Voters",
+    "vote": "Vote",
+    "about": "About",
+    "mustLogin": "You must login",
+    "successfulVote": "Successful vote",
+    "somethingWrong": "Something went wrong, try again"
+  },
+  "about": {
+    "votersMetaTitle": "About",
+    "whatIsEden": "What is Eden Proxy?",
+    "edenDescription": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse maximus molestie laoreet. Vestibulum in varius neque. Praesent at finibus nisi. Vestibulum sagittis nec ligula non vulputate. Donec eu nulla rutrum, volutpat eros sit amet, tempor augue. Maecenas risus quam, efficitur nec enim quis, pretium semper turpis. Duis id arcu fringilla, vulputate nulla eget, lobortis nisi. Cras quis feugiat lectus. Donec vestibulum et dolor a vestibulum. In interdum, ex nec porttitor consequat, nulla tellus malesuada velit, eu pretium augue dolor ut nisi. Proin pretium convallis urna, quis dictum nulla ornare eu. Nulla et mollis lorem. Nulla pretium lacus tortor, sit amet dignissim mi tincidunt non. Suspendisse ut maximus dui. Ut massa mi, luctus a magna ut, porta viverra libero.",
```

Comment:
can we replace this placeholder text with some real text?

Reply:
I think Hahn is working on it!